data/hai/hai reveal 1 intro.txt

```diff
@@ -524,12 +532,26 @@ mission "Hai Leaks Response 1"
 			`	You haven't really seen very much of the Hai so far, and you can't help but think the size of the detachment is rather militant of them. It's impossible to judge if it's for your safety or whether, given your status as a "war hero", it's for theirs.`
 			`	Word of your status in human space also seems to have spread ahead of you, perhaps because you are a significant human visitor in the current climate. A small crowd of Hai gather to peek at you on the short walk from the spaceport to the government building you are led to.`
 			label resume
+			choice
+				`	(Keep silent and observe.)`
+					goto observe
+				`	(Ask about the large security detachment.)`
+			`	The Hai closest to you, who happens to have a very decorated uniform, turns to respond. "It's for crowd control. You've become the center of attention of politicians in a media circus."`
+			`	He looks back to the crowd for a moment. "Life in Hai space is complicated of late, and we want to make sure you are not stopped by crowds with endless questions before reaching your meeting. It would be... inefficient."`
+			choice
+				`	(Keep silent and observe.)`
+				`	"Thank you for your service."`
+					goto thank
+			label observe
 			branch karma
 				karma > 1
 			`	As you are making the walk you hear questions posed and rumors stated. Occasionally you hear words like "survivor" and "dangerous" thrown in amongst the babble.`
 				goto main
 			label karma
 			`	As you are making the walk you hear questions posed and rumors stated. Occasionally you hear words like "honorable" and "reasonable" thrown in amongst the babble.`
```

Comment:
Are the Hai speaking in your language? Or are these humans talking? It's an interesting part, though.

Reply:
It is a Hai talking, but I cannot guess why they're talking in Human Standard. It does seem a bit odd. Either they want you to understand, or they're talking to another human.

Reply:
I changed it to this:

At a table nearby, a Hai says, "Oh, that's awful!" and the human next to her responds, "Feels like being back on Solace."

Solace is a Free Worlds planet near some pirates, waaaay at the far end of human space.